internal/venice-common/src/main/java/com/linkedin/venice/kafka/protocol/enums/ControlMessageType.java

```diff
@@ -24,8 +24,8 @@
  *       not support evolution (i.e.: adding values) properly.
  */
 public enum ControlMessageType implements VeniceEnumValue {
-  START_OF_PUSH(0), END_OF_PUSH(1), START_OF_SEGMENT(2), END_OF_SEGMENT(3), @Deprecated
-  START_OF_BUFFER_REPLAY(4), START_OF_INCREMENTAL_PUSH(5), END_OF_INCREMENTAL_PUSH(6), TOPIC_SWITCH(7);
+  START_OF_PUSH(0), END_OF_PUSH(1), START_OF_SEGMENT(2), END_OF_SEGMENT(3), START_OF_INCREMENTAL_PUSH(4),
+  END_OF_INCREMENTAL_PUSH(5), TOPIC_SWITCH(6), VERSION_SWAP(7);
 
```

Comment:
The numbers must stay the same across all versions of the software. We cannot change them.